robot-custom-integration/sl_python_robot/SLListener.py

```diff
@@ -35,11 +55,27 @@ def end_suite(self, date, result):
         self.send_test_results(test_results)
         self.end_test_session()
 
-    #--- Sealights API helpers ---
-    
+    def start_test(self, data, result):
+        test_name = data.name
```

Comment:
We need to make sure `test_name` is URL encoded in case of white spaces etc..

Reply:
I will take care of this in the OTEL plugin in the browser-agent, will encode always the testName if not already

Reply:
@kristijanstefanoski , done here.
https://sealights.atlassian.net/browse/SLDEV-14387